crates/ruma-common/Cargo.toml

```diff
@@ -66,6 +67,9 @@ url = "2.2.2"
 uuid = { version = "0.8.2", optional = true, features = ["v4"] }
 wildmatch = "2.0.0"
 
+[target.'cfg(all(target_arch = "wasm32", target_os = "unknown"))'.dependencies]
+js-sys = { version = "0.3", optional = true }
```

Comment:
@DevinR528 cargo-sort complains about this not being sorted, is this a known bug?

Reply:
Well this *is* the `target` table (with sub-tables). I just didn't look at the error message carefully and assumed this would be about the entries of the table. I forgot `cargo-sort` also enforces ordering of the tables themselves 🙂

Reply:
Oh, the error message even says dependencies. Well that should be improved then!